app/views/unattended/provisioning_templates/PXEGrub2/autoyast_default_pxegrub.erb

```diff
@@ -0,0 +1,23 @@
+<%#
+kind: PXEGrub
+name: AutoYaST default PXEGrub
+model: ProvisioningTemplate
+oses:
+- SLES
+- OpenSUSE
+description: |
+  The template to render PXEGrub bootloader configuration for AutoYaST based distributions.
+  The output is deployed on the host's subnet TFTP proxy.
+-%>
+# This file was deployed via '<%= template_name %>' template
+
+default=0
+timeout=<%= host_param('loader_timeout') || 10 %>
+<% http_proxy_string = host_param('http-proxy') ? "proxy=http://" + host_param('http-proxy') + ":" + host_param('http-proxy-port') : '' -%>
+
+title <%= template_name %>
+  root (nd)
+  kernel (nd)/../<%= @kernel %> ramdisk_size=65536 install=<%= @mediapath %> autoyast=<%= foreman_url('provision') %> textmode=1 <%= http_proxy_string %>
+  initrd (nd)/../<%= @initrd %>
+
+<%= snippet_if_exists(template_name + " custom menu") %>
```

Comment:
Wrong directory, have you tested this? Grub1 is rarely used, we have it for RHEL6 provisioning but this is old stuff.

Reply:
Moved to the correct directory

As for testing: my focus was on the Grub2 template but I can get caught up with a test of this one as well